python/xorbits/_mars/dataframe/merge/merge.py

```diff
@@ -115,11 +115,28 @@ def execute_map(cls, ctx, op):
         for index_idx, index_filter in enumerate(filters):
             reducer_index = (index_idx, chunk.index[1])
             if index_filter is not None and index_filter is not list():
```

Comment:
I didn't notice this line before, but is `index_filter is not list()` correct?

Reply:
Fixed. I found that the entire judgement "if index_filter is not None and index_filter is not list():" is always True since the ``index_filter`` is always a list. So I clean these codes here.